health/health.d/bcache.conf

```diff
@@ -22,8 +22,7 @@ component: Disk
      calc: $dirty + $metadata + $undefined
     units: %
     every: 1m
-     warn: $this > ( ($status >= $WARNING ) ? ( 70 ) : ( 90 ) )
-     crit: $this > ( ($status == $CRITICAL) ? ( 90 ) : ( 95 ) )
+     warn: $this > 75
```

Comment:
I think this is not really needed but I have no problem with restoring it back. This just simplifies the whole thing

Reply:
>  the original reason is for that, just wondering

I think there is one reason for using [conditional operator](https://learn.netdata.cloud/docs/alerting/health-configuration-reference#special-use-of-the-conditional-operator):

>  This usage helps to avoid bogus messages resulting from **small variations in the value when it is varying regularly but staying close to the threshold value**, without needing to delay sending messages at all.

It is recommended to use it for every value like this (e.g. temperature)